### DISCUSSION TITLE
Improve current profile settings in Fans + Power section

### DISCUSSION THREAD

**Discussion by CalinBedea** [seerge/g-helper#2727] [Q&A]
Hello! I recently installed Ghelper to boost battery life on my laptop, but found many more interesting settings. I did a bit of research, since I was not accustomed with some concepts (like undervolting), and applied what I think would work best. I'm not sure if the modified settings are ok, so it would be much appreciated if I could get some tips on how to improve them. 

On silent mode, I want the best possible battery life, since the laptop will not be plugged in - I will be using it only on my browser, looking at PDFs, YouTube videos, etc. I have tried undervolting it a bit: from 2-3h it went to 8h! This was a game-changer.

On balanced and turbo, it will be plugged in only. The balanced profile I will be using most of the time, and the turbo profile only while gaming.

For context, I have an ASUS A16 2024 with Ryzen 9 7845HX and RTX 4060. I don't want to damage any hardware or something, that's why I came here to ask for tips. It's a brand new laptop and I don't want to ruin its lifespan. Thank you for designing this great app! I will provide screenshots with my current settings.

![Screenshot 2024-06-13 130741](https://github.com/seerge/g-helper/assets/75206353/7a89299c-252d-419c-834a-0c599e9252c1)

![Screenshot 2024-06-13 130658](https://github.com/seerge/g-helper/assets/75206353/32bf3cdb-a300-4cba-8eee-c20d5b119168)

![Screenshot 2024-06-13 130812](https://github.com/seerge/g-helper/assets/75206353/8f06da8f-b04d-4446-81ea-1d947d7463c0)

![Screenshot 2024-06-13 130841](https://github.com/seerge/g-helper/assets/75206353/8b2c75d2-aafa-4279-8fbd-1391132276dc)

![Screenshot 2024-06-13 130829](https://github.com/seerge/g-helper/assets/75206353/73d7be17-4cda-4073-8d4f-33c6eb23803d)

![Screenshot 2024-06-13 130850](https://github.com/seerge/g-helper/assets/75206353/2757fde7-ddac-49ef-ab6b-47446ae5c50a)

![Screenshot 2024-06-13 130902](https://github.com/seerge/g-helper/assets/75206353/2b95f9c2-299a-4f37-a843-5bde8363fd1e)

**Comment by seerge** [comment]
@CalinNebunu hello. Usually in most of the cases default profiles (stored in BIOS) are absolutely fine for every day usage. So there is no need to set custom power limits / etc.

If you pursue specific goals - you can tweak them to your needs of course. As for undervolting - if it runs stable with this values for you - then you are good.
